jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/domain/mappers/ForecastDataMapper.kt

```diff
@@ -9,19 +9,34 @@ import com.e_conomic.weatherapp.domain.model.Forecast as ModelForecast
 
 class ForecastDataMapper {
 
-    fun convertFromDataModel(forecast: ForecastResult): ForecastList {
+    companion object {
+        val NO_ICON = "ForecastDataMapper:noIcon"
+        val NO_DESCRIPTION = "ForecastDataMapper:noDescription"
```

Comment:
Also, I think that is best practice, as setting the magic `ForecastDataMapper:noDescription` as the description, makes it impossible for the user to set that as a description...